### PR TITLE
Fixed duplicated provider that was exhibited in json response

### PR DESCRIPTION
**Pull request by defong**
### Context
Duplicated provider that was exhibited in json response

### Changes proposed in this pull request
Fixed duplicated provider that was exhibited in json response

### CODE REVIEW

**Comment by defong** [comment]
[Duplicate providers in the incremental Provider API](https://trello.com/c/Nn9OGiBP/938-duplicate-providers-in-the-incremental-provider-api)

**Comment by benilovj** [comment]
The spec is very verbose. One of the major benefits of `factory_bot` is to be able to skip the parts of the test data that you don't care about and let `factory_bot` provide sensible defaults. Is there a way to strip the spec down to the actual attributes that are relevant for the particular scenario?

**Comment by defong** [comment]
Yeah the factory can set most of it, I was verbose, using data from endpoint.

**Comment by defong** [comment]
> > Yeah the factory can set most of it, I was verbose, using data from endpoint.
> 
> OK, could we strip the spec back to just the things that trigger the bug, so the clarity isn't lost? smiley

Sorted, as far as I can, not sure of the https://github.com/DFE-Digital/manage-courses-backend/blob/819c6fa18f59658e980f10b8cdf0d323ce610b70/spec/factories/providers.rb#L40

`age` is passed around its looks like a vintage singleton value, so sticking with hard coded dates.

File: spec/requests/api/v1/provider_spec.rb [hunk 1]
```diff
@@ -290,6 +290,62 @@ def get_provider_codes_from_body(body)
...
+                postcode: 'enrichment1 postcode',
+                region_code: 6,
+                created_at: "2019-02-05T10:32:14.456414",
+                updated_at: "2019-02-05T10:32:22.95923",
+                status: 1)
+        end
+        let(:old_published_enrichment) do
+          build(:provider_enrichment,
+                region_code: 6,
+                created_at: "2018-09-26T15:25:33.191305",
```
Comment (dankmitchell) [inline spec/requests/api/v1/provider_spec.rb]: Might be easier to read using a date object i.e. `Date.yesterday`

Comment (defong) [reply spec/requests/api/v1/provider_spec.rb]: I was using the exact dates that was showing the bug.

**Pull request merged**